foodnet/membership/tests.py

```diff
@@ -35,15 +47,15 @@ def setUp(self):
         email_address.save()
 
     def test_profile(self):
-        resp = self.client.get(reverse('profile'))
+        response = self.client.get(reverse('profile'))
```

Comment:
For some reason you believe that `resp` is less readable than `response` in tests that have 4-5 lines. :-)

To me it looks you want us to adopt your guidelines not just some guidelines. We review each other's code so if something really need refactoring I'm sure it will be raised in PR. Everyone knows the argument for code readability. We don't need to have policies (yet another document to read) which tries to enforce that.